docs/sql-ref-syntax-aux-resource-mgmt-add-jar.md

```diff
@@ -33,15 +33,30 @@ ADD JAR file_name
 
 * **file_name**
 
-    The name of the JAR file to be added. It could be either on a local file system or a distributed file system.
+    The name of the JAR file to be added. It could be either on a local file system or a distributed file system or an ivy URL.
+    Apache Ivy is a popular dependency manager focusing on flexibility and simplicity. Now we support two parameter in URL query string:
 
+     * transitive: whether to download dependent jars related to your ivy URL.
```

Comment:
"It is case-sensitive and only take last one if multiple transitive parameters are specified."

Reply:
> "It is case-sensitive and only take last one if multiple transitive parameters are specified. Invalid value will be treated as false."

Done and update snapshot in desc.